packages/plasma-new-hope/src/components/Popup/PopupContext.tsx

```diff
@@ -21,7 +21,7 @@ const PopupContext = createContext<PopupContextType>({
 export const usePopupContext = () => useContext(PopupContext);
 
 export const PopupProvider: FC<PropsWithChildren> = ({ children }) => {
-    const prevBodyOverflowY = useRef(document.body.style.overflowY);
+    const prevBodyOverflowY = useRef(typeof document !== 'undefined' ? document.body.style.overflowY : '');
```

Comment:
Так нельзя? Я просто хз какой document вообще может быть (или может ли он быть null к примеру)
```suggestion
    const prevBodyOverflowY = useRef(document ? document.body.style.overflowY : '');
```

Reply:
нет, так мы обращаемся к переменной

Reply:
А typeof не обращается к переменной?

Reply:
Typeof не ругается, когда вар не определен. Странно, я не знал этого) апрувнул

Reply:
`typeof sometest`

Reply:
ты пропустил времена jquery? )